src/dispatch_package/dispatch.cpp

```diff
@@ -72,19 +76,19 @@ struct ClosestDepotInfo {
 
 
 struct Parameters {
-  double battery_cap_kwh         = 200.0; //kW*hr
-  double kwh_per_km              = 1.2;   //kW*hr/km
-  double charging_rate           = 150;   //kW
-  double search_radius           = 1000;  //m //TODO
-  double zstops_frac_stopped_at  = 0.2;
-  double zstops_average_time     = 10;    //seconds
+  kilowatt_hours battery_cap_kwh        = 200.0_kWh;       //kW*hr
```

Comment:
does the units package interpret the suffix on `kwh_per_km` as units? If so, why no units for all the others?

Reply:
The others are either unitless quantities or, in the case of the charging rate, unused, and will be pulled out.